test/controllers/update/update.use.paper.controller.spec.ts

```diff
@@ -28,4 +29,13 @@ describe("UPDATE USE PAPER controller", () => {
       expect(resp.text).toContain(UPDATE_APPLY_PAPER_FORM_HEADING);
     });
   });
+
+  test(`renders the ${config.USE_PAPER_PAGE} page for remove`, async () => {
+    const resp = await request(app).get(`${UPDATE_USE_PAPER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`);
+
+    expect(resp.status).toEqual(200);
+    expect(resp.text).toContain(UPDATE_APPLY_PAPER_FORM_HEADING);
```

Comment:
you could also check the banner text to expect if it is the 'remove' version and add a similar check into the existing test above to check that it has the 'update' banner text